python_scripts/ensemble_hyperparameters.py

```diff
@@ -62,8 +62,17 @@
 
 # %% [markdown]
 # We can observe that in our grid-search, the largest `max_depth` together
-# with the largest `n_estimators` led to the best generalization performance.
-#
+# with the largest `n_estimators` led in average to the best performance
+# on the validation sets. Now we will estimate the generalization performance
+# of the best model by refitting it with the full training set and using the test
```

Comment:
I assume this is the discussion that we had at launch IRL. Here, we need a rephrasing because it seems that the refitting is done in `score`. However, this is done only in the previous call to `fit`.

Reply:
You are right, I forgot to edit this notebook after our discussion, Thanks!